src/client/components/app-core/form.css

```diff
@@ -0,0 +1,33 @@
+label {
+  cursor: pointer;
+}
+
+[type="text"],
+[type="email"],
+[type="password"] {
+  padding-top: var(--spacing-half);
+  padding-bottom: var(--spacing-half);
+  padding-left: var(--spacing-default);
+  padding-right: var(--spacing-default);
```

Comment:
have you heard of shorthand properties? ;) why not: `padding: var(--spacing-half) var(--spacing-default);`

Reply:
`true`